models.py

```diff
@@ -82,7 +86,7 @@ def __init__(self, uuid, ip_range):
 
     def save(self):
         db.insert('ip_range',institution_uuid=self.institution_id,
-                ip_range_value=self.ip_range)
+                ip_range_value=self.ip_range) # TODO what if it's an array?
```

Comment:
This class represent the object IPRange - if there are multiple ranges given each of them will be their own instance of IPRange